lib/screens/dji_shop/screens/product_screen/widgets/product_description.dart

```diff
@@ -0,0 +1,219 @@
+              SizedBox(height: 10),
+              _Quantity(),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _ProductName extends StatelessWidget {
+  const _ProductName({
+    Key? key,
+    required this.productName,
+  }) : super(key: key);
+  final String productName;
+  @override
+  Widget build(BuildContext context) {
+    return Text(
+      productName,
+      style: context.theme.textTheme.headline5!.copyWith(
+        fontWeight: FontWeight.bold,
+      ),
+    );
+  }
+}
+
+class _RatingReviewPrice extends StatelessWidget {
+  const _RatingReviewPrice({
+    Key? key,
+    required this.review,
+    required this.price,
+    required this.rating,
+  }) : super(key: key);
+
+  final String review;
+  final String price;
+  final double rating;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Row(
+          children: [
+            RatingBar.builder(
+              itemSize: 20,
+              initialRating: rating,
+              minRating: 1,
+              direction: Axis.horizontal,
+              allowHalfRating: true,
+              itemCount: 5,
+              itemPadding: const EdgeInsets.symmetric(horizontal: 4.0),
+              itemBuilder: (context, _) => const Icon(
+                Icons.star,
+                color: Colors.amber,
+              ),
+              onRatingUpdate: (rating) {},
+            ),
+            const SizedBox(width: 5),
+            Text(
+              review,
+              style: context.theme.textTheme.button!.copyWith(
+                color: Colors.grey,
+              ),
+            ),
+          ],
+        ),
+        Text(
+          price,
+          style: context.theme.textTheme.headline6!.copyWith(
+            color: Colors.black,
+            fontWeight: FontWeight.bold,
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class _DescriptionProduct extends StatelessWidget {
+  const _DescriptionProduct({
+    Key? key,
+    required this.description,
+  }) : super(key: key);
+  final String description;
+
+  @override
+  Widget build(BuildContext context) {
+    return Text(
+      description,
+      style: context.theme.textTheme.subtitle1!.copyWith(
+        color: Colors.black,
+        height: 1.5,
+      ),
+    );
+  }
+}
+
+class _Quantity extends StatelessWidget {
+  const _Quantity({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Text(
+          'Quantity',
+          style: context.theme.textTheme.subtitle1!.copyWith(
+            color: Colors.grey,
+          ),
+        ),
+        Row(
+          children: [
+            InkWell(
```

Comment:
move to another widget which can be used for increment and decrement (for adding and removing quantity of drones)